apps/docs/content/docs/frameworks/remix.mdx

```diff
@@ -17,9 +17,109 @@ Requirements:
 
 To use NextUI in your Remix project, you need to follow the following steps:
 
+### Using NextUI + Remix template
+
+If you are starting a new project, you can run one of the following commands to create a Remix project pre-configured with NextUI:
+
+<PackageManagers
+  commands={{
+    npm: 'npx create-remix@latest --template https://github.com/nextui-org/remix-template',
```

Comment:
TODO: 
- [ ] Making the remix template public.

Reply:
done